### DISCUSSION TITLE
Horizontal Collapse or Accordion

### DISCUSSION THREAD

**Discussion by shlitaTeam** [mantinedev/mantine#6287] [Q&A]
### Link to the page where something is not right

https://v7.mantine.dev/core/collapse

### What is wrong?

is there way to use Collapse or Accordion in Horizontaly?
the item will be opened from the side (left or right) and not bottom.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
No, Mantine does not provide such option.

**Reply by shlitaTeam** [reply]
is there any other solution that you can suggests for this case?
